astropy/convolution/tests/test_convolve_fft.py

```diff
@@ -69,6 +70,18 @@ def assert_floatclose(x, y):
 
 class TestConvolve1D:
 
+    def test_quantity(self):
```

Comment:
Does this need a `@pytest.mark.parametrize(option_names, options)` like the other tests?